lib/web3/service.ts

```diff
@@ -75,10 +76,12 @@ class Web3Service {
     data: string,
     address: string
   ): Promise<SignResult | null> {
+    let signer: any;
     try {
       const web3 = this.web3Instance;
       const hashedData = web3.utils.sha3(data);
-      const signature = await web3.eth.personal.sign(hashedData, address, "");
+      signer = LocalStorage.getItem("ongama_signer_address");
+      const signature = await web3.eth.personal.sign(hashedData!, signer, "");
```

Comment:
Why is the third argument empty?

Reply:
i wasn't able to find out what to feel in, so, i'm trying  to reach out to @danmugh  so, he can let me know what to pass as third argument

Reply:
Awesome!

Let me know when it's done.

Reply:
okay sir